Makefile.pre.in

```diff
@@ -418,6 +418,8 @@ PYTHON_OBJS=	\
 		Python/frozenmain.o \
 		Python/future.o \
 		Python/gc.o \
+		Python/gc_free_threading.o \
+                Python/gc_gil.o \
```

Comment:
Can you fix the indentation here? :)

Reply:
@erlend-aasland To how?

Reply:
I took the liberty to fix it in 0d8dc3df929d768470572a709e52bc9f15342a28 :) Make uses tabs for a lot of the indenting. This list of source files is indented by two tabs, not eight spaces.

Reply:
Ah, I always use softtab, accidents always happen :)